### PR TITLE
Ch final

### PR DESCRIPTION
**Pull request by aac-devs**
captura final

### CODE REVIEW

File: database-debug.log [hunk 1]
```diff
@@ -0,0 +1,7 @@
+WARNING: An illegal reflective access operation has occurred
+WARNING: Illegal reflective access by io.netty.util.internal.ReflectionUtil (file:/home/aac-devs/.cache/firebase/emulators/firebase-database-emulator-v4.7.2.jar) to field sun.nio.ch.SelectorImpl.selectedKeys
+WARNING: Please consider reporting this to the maintainers of io.netty.util.internal.ReflectionUtil
+WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
+WARNING: All illegal access operations will be denied in a future release
+13:46:32.737 [NamespaceSystem-akka.actor.default-dispatcher-5] INFO akka.event.slf4j.Slf4jLogger - Slf4jLogger started
+13:46:33.288 [main] INFO com.firebase.server.forge.App$ - Listening at localhost:9000
```
Comment (dianaatwizeline) [inline database-debug.log]: this file can be omitted from the PR

File: ui-debug.log [hunk 2]
```diff
@@ -0,0 +1,73 @@
+Web / API server started at localhost:4000
```
Comment (dianaatwizeline) [inline ui-debug.log]: this file can also be omitted

**Review by dianaatwizeline — COMMENTED** [review]
As with every deliverable, really good job.  I love how your app looks and feels

**Comment by aac-devs** [comment]
Thank you very much for your comments, I am very pleased that you liked my app.

**Pull request merged**